client/src/components/ElementEditor/tests/Header-test.js

```diff
@@ -1,307 +1,217 @@
-          onDragEnd={() => {}}
-        />
-      );
-
-      const tooltip = wrapper.find('Tooltip');
-      expect(tooltip.length).toBe(0);
-    });
-
-    it('should render a "right caret" button when not expandable', () => {
-      const wrapper = shallow(
-        <Header
-          element={element}
-          areaId={1}
-          type={type}
-          expandable={false}
-          ElementActionsComponent={ElementActionsComponent}
-          connectDragSource={content => content}
-          connectDragPreview={content => content}
-          onDragEnd={() => {}}
-        />
-      );
-
-      const expandButton = wrapper.find('.element-editor-header__expand');
-      expect(expandButton.length).toBe(1);
-      expect(expandButton.hasClass('font-icon-right-open-big')).toBe(true);
-    });
-
-    it('should render a "down caret" button when not expanded', () => {
-      const wrapper = shallow(
-        <Header
-          element={element}
-          areaId={1}
-          type={type}
-          expandable
-          previewExpanded={false}
-          ElementActionsComponent={ElementActionsComponent}
-          connectDragSource={content => content}
-          connectDragPreview={content => content}
-          onDragEnd={() => {}}
-        />
-      );
+}
 
-      const expandButton = wrapper.find('.element-editor-header__expand');
-      expect(expandButton.length).toBe(1);
-      expect(expandButton.hasClass('font-icon-down-open-big')).toBe(true);
-    });
-
-    it('should render an "up caret" button when expanded', () => {
-      const wrapper = shallow(
-        <Header
-          element={element}
-          areaId={1}
-          type={type}
-          expandable
-          previewExpanded
-          ElementActionsComponent={ElementActionsComponent}
-          connectDragSource={content => content}
-          connectDragPreview={content => content}
-          onDragEnd={() => {}}
-        />
-      );
-
-      const expandButton = wrapper.find('.element-editor-header__expand');
-      expect(expandButton.length).toBe(1);
-      expect(expandButton.hasClass('font-icon-up-open-big')).toBe(true);
-    });
+test('Header should render the icon', () => {
+  const { container } = render(<Header {...makeProps()}/>);
+  expect(container.querySelector('i.font-icon-block-file')).not.toBeNull();
+});
 
-    it('should not render a "caret" button for a broken element', () => {
-      const wrapper = shallow(
-        <Header
-          element={element}
-          areaId={1}
-          type={typeBroken}
-          expandable={false}
-          ElementActionsComponent={ElementActionsComponent}
-          connectDragSource={content => content}
-          connectDragPreview={content => content}
-          onDragEnd={() => {}}
-        />
-      );
+test('Header should render the title', () => {
+  const { container } = render(
+    <Header {...makeProps({
+      element: {
+        id: '12',
+        title: 'Sample File Block'
+      }
+    })}
```

Comment:
```suggestion
    <Header {...makeProps()}
```
The ID doesn't seem important (you're not checking the ID of the element) and the title is the same as the default.